src/IO/read_input.jl

```diff
@@ -145,10 +145,16 @@ maxpax = readmis("max_payload_in_pax_equivalent") #This represents the maximum a
 pax = readmis("pax")
 exitlimit = readmis("exit_limit")
 despax = pax[1] #Design number of passengers
-if despax > maxpax
-    error("Design mission has higher payload weight than maximum aircraft payload!")
-elseif despax > exitlimit
-    error("Design mission has higher # of passengers than the exit limit!")
+
+if any(maxpax .< pax)
+    error("One or more missions have higher payload than prescribed maximum aircraft payload!:"*
+    "\n Max Payload [in pax] = "*string(maxpax)*
+    "\n Payloads listed [in pax] = "*string(pax))
+end
+if any(exitlimit .< pax)
+    error("One or more missions have higher payload than the prescribed exit limit!:"*
```

Comment:
Maybe say "One or more missions have number of passengers than the prescribed exit limit"